package.json

```diff
@@ -74,6 +73,7 @@
     "@artsy/stitch": "6.1.6",
     "@artsy/xapp": "1.0.6",
     "@loadable/component": "5.12.0",
+    "@loadable/server": "5.13.2",
```

Comment:
What does this package bring? I don't see it required by any files, maybe also requires its own pr if unrelated to storybooks.

Reply:
https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/buildServerApp.tsx#L31